app/shared/components/turnstileWidget/TurnstileWidget.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
+
+import { Locale } from 'next-intl';
+import Turnstile from 'react-cloudflare-turnstile';
+interface TurnstileWidgetProps {
+  onSuccessAction: (token: string) => void;
+  language: Locale;
+}
+
+export default function TurnstileWidget({ onSuccessAction, language }: Readonly<TurnstileWidgetProps>) {
+  return (
+    <div className="mt-4">
```

Comment:
```suggestion
    <Box mt={4}>
```
Maybe it will be better to use MUI's Box instead of div

Reply:
I agree with you, thanks